Modules/+Experiments/+AutoExperiment/@SpecSlowScan/SpecSlowScan.m

```diff
@@ -1,6 +1,6 @@
 classdef SpecSlowScan < Experiments.AutoExperiment.AutoExperiment_invisible
-    %SpecSlowScan Automatically performs 1) spectra, 2) open-loop PLE, and
-    %3) closed-loop PLE on identified sites
+    %SpecSlowScan Automatically performs 1) spectra, 2) open-loop PLE,
+    %3) closed-loop PLE 4) and SuperRes on identified sites
```

Comment:
Reasoning for adding this? Seems like its own thing.

Reply:
it's part of the optical stack